src/router.js

```diff
@@ -13,12 +15,22 @@ const routes =[
         name: 'contractor-home',
         component: ContractorHome,
     } ,
+    {
+        path: '/bidderHome',
+        name: 'bidder-home',
```

Comment:
try to follow this approach 
<img width="827" alt="image" src="https://user-images.githubusercontent.com/16922245/225571658-4aee1d53-e2d0-4434-a70b-e976927d4b07.png">


https://vueschool.io/articles/vuejs-tutorials/how-to-structure-a-large-scale-vue-js-application/